mypy.ini

```diff
@@ -63,3 +63,7 @@ disallow_untyped_defs = False
 ignore_missing_imports = True
 [mypy-flask.signals]
 ignore_missing_imports = True
+[mypy-starlette.*]
+ignore_missing_imports = True
+[mypy-fastapi.*]
+ignore_missing_imports = True
```

Comment:
Are these really needed? In particular the FastAPI one. If there are mypy errors in FastAPI I haven't seen I would like to get them fixed!

Reply:
Those are there because the integrations import stuff (aka fastapi) that is outside of the sentry sdk in the users projects.

And i basically just was to lacy to set mypy up correctly so these ignores are not necessary. But I will do so know :-)

Reply:
done.